src/components/FloatingBox/TabletPopup.tsx

```
@@ -1,15 +1,28 @@
+'use client';
+
+import useBookingStore from '@/stores/Booking/useBookingStore';
+import IconClose from '@assets/svg/close';
 import DatePicker from '../DatePicker/DatePicker';
+import TimeSelector from './TimeSelector';
+
+export default function TabletPopup({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)